js/index.js

```diff
@@ -1,7 +1,46 @@
 // Iteration 1: Names and Input
-
+let hacker1="Elia"
+console.log(`The driver's name is ${hacker1}`)
+let hacker2="ceyda"
+console.log(`The navigator's name is ${hacker2}`)
 
 // Iteration 2: Conditionals
+if (hacker1.length > hacker2.length){
+    console.log(`The driver has the longest name, it has ${hacker1.length} characters.`)
+  } else if (hacker1.length<hacker2.length){
```

Comment:
Please be aware of both indentation and spacing on this line. Same applies for the following lines.